src/OpenConext/Profile/Value/ConsentList.php

```diff
@@ -35,11 +35,14 @@
     public function __construct(array $consents)
     {
         foreach ($consents as $consent) {
-            $this->initialiseWith($consent);
+            $this->initializeWith($consent);
         }
     }
 
-    private function initialiseWith(Consent $consent)
+    /**
+     * @param Consent $consent
+     */
+    private function initializeWith(Consent $consent)
```

Comment:
I typically don't write a docblock if it doesn't have any added value. No need to remove it, but explaining my motives for not having written one :wink: